doc/classes/AnimationNode.xml

```diff
@@ -152,13 +152,25 @@
 				Gets the value of a parameter. Parameters are custom local memory used for your animation nodes, given a resource can be reused in multiple trees.
 			</description>
 		</method>
+		<method name="get_processing_animation_tree_instance_id" qualifiers="const">
+			<return type="int" />
+			<description>
+				Returns the object id of the [AnimationTree] that owns this node. This method should only be called from within the [method AnimationNodeExtension._process] method, and will return an invalid id otherwise.
```

Comment:
```suggestion
				Returns the object id of the [AnimationTree] that owns this node.
				[b]Note:[/b] This method should only be called within [method AnimationNodeExtension._process]. Otherwise, this method returns an invalid id.
```